app/src/pages/inside/logsPage/logItemInfo/infoTabs/infoTabs.jsx

```diff
@@ -41,6 +42,8 @@ export class InfoTabs extends Component {
     return activeTab && tab.id === activeTab.id;
   }
 
+  isOnMobile = () => window.matchMedia(SCREEN_XS_MAX).matches;
```

Comment:
Please use listeners to support screen resizing

Reply:
Fixed